server/auth/login.py

```diff
@@ -38,7 +40,16 @@ def login_request(self, packet, address):
             auth_status = 'bad_credentials'
             uid = None
         else:
-            uid = self.server.handle_until_value('auth:get_user_id', True, address)
+            user = authenticate(username=packet.username, password=packet.password)
+
+            uid = user.id
+
+            account = self.server.handle_until_return('auth:get_game_account', uid)
+
+            if not account:
+                account = self.server.handle_until_value('auth:create_game_account', True, packet.username, packet.password, address)
```

Comment:
Registration should be via the CMS, once that gets set up. This is insufficient for registration.